test/wasmBenchmarker/benchmark.py

```diff
@@ -281,11 +386,24 @@ def main():
     emcc_path = get_emcc(args.verbose, not args.no_system_emcc)
     test_names = compile_tests(emcc_path, args.test_dir, args.only_game, args.only_simd, args.compile_anyway, args.run, args.verbose)
 
-    result_data = run_tests(args.test_dir, test_names, args.engines, args.iterations, args.mem, args.jit, args.verbose)
-    generate_report(result_data["time"], args.report)
+    result_data = run_tests(args.test_dir, test_names, args.engines, args.iterations, args.mem, args.no_time, args.jit, args.jit_no_reg_alloc, args.interpreter, args.verbose)
+    if not args.no_time:
+        compare(result_data["time"], args.engines, args.compare_jit_to_interpreter, args.compare_jit_no_reg_alloc_to_interpreter, args.compare_jit_no_reg_alloc_to_jit)
+        print("TIME RESULTS")
+        generate_report(result_data["time"], args.report)
     if (args.mem):
+        compare(result_data["mem"], args.engines, args.compare_jit_to_interpreter, args.compare_jit_no_reg_alloc_to_interpreter, args.compare_jit_no_reg_alloc_to_jit)
+        print("MEMORY RESULTS")
         generate_report(result_data["mem"], memreport)
+    
+    print(errorList)
 
```

Comment:
Perhaps this line could be removed

Reply:
The empty line or the `print(errorList)`?

Reply:
I think there were 2 empty lines. One is enough